packages/gatsby/src/services/index.ts

```diff
@@ -1,16 +1,60 @@
-export { startWebpackServer } from "./start-webpack-server"
```

Comment:
:facepalm: I feel re-exports ought to create local bindings regardless :(

Now that I look at it, what's the thought behind re-exporting all these services rarther than directly importing them in the file where they're used?

Reply:
I prefer shorter imports! But they're also exported as a service map, so they need to be in scope here anyway.